### PR TITLE
Adding iPhone11,2 + iPhone11,4 + iPhone11,8

### PR DESCRIPTION
**Pull request by Coeur**
source: Xcode 10 GM Simulators

### CODE REVIEW

File: DeviceList.plist [hunk 1]
```diff
@@ -198,6 +198,27 @@
 		<key>name</key>
 		<string>iPhone X</string>
 	</dict>
+	<key>iPhone11,2</key>
+	<dict>
+		<key>version</key>
+		<real>11.2</real>
+		<key>name</key>
+		<string>iPhone XS</string>
```
Comment (armandzwan) [inline DeviceList.plist]: Apple calls it iPhone Xs.

File: DeviceList.plist [hunk 2]
```diff
@@ -198,6 +198,27 @@
...
+	</dict>
+	<key>iPhone11,4</key>
+	<dict>
+		<key>version</key>
+		<real>11.4</real>
+		<key>name</key>
+		<string>iPhone XS Max</string>
```
Comment (armandzwan) [inline DeviceList.plist]: Apple calls it Xs Max

File: DeviceList.plist [hunk 3]
```diff
@@ -198,6 +198,27 @@
...
+	</dict>
+	<key>iPhone11,8</key>
+	<dict>
+		<key>version</key>
+		<real>11.8</real>
+		<key>name</key>
+		<string>iPhone XR</string>
```
Comment (armandzwan) [inline DeviceList.plist]: Apple calls it Xr

**Comment by armandzwan** [comment]
I see Watch series 3 also missing. Will try to make a pull request for that.
https://everymac.com/ultimate-mac-lookup/?search_keywords=Watch3,1
https://everymac.com/ultimate-mac-lookup/?search_keywords=Watch3,2
https://everymac.com/ultimate-mac-lookup/?search_keywords=Watch3,3

**Comment by Coeur** [comment]
@armandzwan Xcode disagrees with your naming source:
screen shot 2018-09-13 at 10 08 51

**Comment by armandzwan** [comment]
Then it inconsistent with what they say on their own website:
schermafbeelding 2018-09-13 om 10 25 43

**Comment by Coeur** [comment]
@armandzwan I believe they use [Small caps](https://en.wikipedia.org/wiki/Small_caps) on the website.

X<sub>S</sub> ≠ Xs
X<sub>R</sub> ≠ Xr

**Comment by armandzwan** [comment]
They can't make up their mind. :-) Let's stick to XS then, like XCode says.

**Comment by mtak-** [comment]
Any movement on this?

**Comment by mtak-** [comment]
We have a `XS Max` in the office, and it's definitely `11,6`

**Comment by Coeur** [comment]
Not an official source, but according to Hannes Sverrisson on https://stackoverflow.com/a/52412893/1033581:
>    // 2018 models
    if ([platform isEqualToString:@"iPhone11,1"])    return @"iPhone XS (GSM+CDMA)";
    if ([platform isEqualToString:@"iPhone11,2"])    return @"iPhone XS (GSM)";
    if ([platform isEqualToString:@"iPhone11,4"])    return @"iPhone XS Max (GSM+CDMA)";
    if ([platform isEqualToString:@"iPhone11,6"])    return @"iPhone XS Max (GSM)";
    if ([platform isEqualToString:@"iPhone11,8"])    return @"iPhone XR (GSM+CDMA)";
    if ([platform isEqualToString:@"iPhone11,9"])    return @"iPhone XR (GSM)";

We could eventually extrapolate to 11,1/2/3 being XS variants, 11,4/5/6 being XS Max variants and 11,7/8/9 being XR variants.

**Comment by Coeur** [comment]
A kind of reliable source is https://www.theiphonewiki.com/wiki/Models, and it lists (alphabetically):
* iPhone XR is iPhone11,8
* iPhone XS is iPhone11,2
* iPhone XS Max is iPhone11,4
* iPhone XS Max CHINA is iPhone11,6

@mtak- was your office XS Max device bought in China by any chance?

**Comment by mtak-** [comment]
@Coeur nope. it's intended for AT&T customers here in the states.

**Comment by Coeur** [comment]
Then, it's quite annoying that every source we have on the internet give conflicting informations, and Apple keeps it secret.

**Comment by mtak-** [comment]
if you're like us and can't keep waiting, we have a fork here: https://github.com/BrainiumLLC/DeviceUtil
and a pod `pod install DeviceUtil_Brainium`

**Comment by InderKumarRathore** [comment]
Merged

**Comment by InderKumarRathore** [comment]
Sorry guys for the delay, I didn't have my laptop with me. I really appreciate your efforts to make this library upto date  👏

**Pull request merged**